pkg/podmanager/pod_manager.go

```diff
@@ -238,3 +271,161 @@ func (pm *podManager) GetPodTopController(ctx context.Context, pod *corev1.Pod)
 		UID: podOwner.UID,
 	}, nil
 }
+
+// Default implements admission.CustomDefaulter.
+func (pw *podManager) Default(ctx context.Context, obj runtime.Object) error {
+	// Avoids affecting the time of pod creation when dra is not enabled
+	if !pw.enableDra {
+		return nil
+	}
+
+	logger := logutils.FromContext(ctx)
+	pod := obj.(*corev1.Pod)
+	mutateLogger := logger.Named("Mutating").With(
+		zap.String("Pod", pod.Name))
+	mutateLogger.Sugar().Debugf("Request Pod: %+v", *pod)
+
+	return pw.injectPodResources(logutils.IntoContext(ctx, mutateLogger), mutateLogger, pod)
+}
+
+func (pw *podManager) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
+	return nil, nil
+}
+
+func (pw *podManager) ValidateUpdate(ctx context.Context, oldObj, newObj runtime.Object) (admission.Warnings, error) {
+	return nil, nil
+}
+
+// ValidateDelete will implement something just like kubernetes Foreground cascade deletion to delete the MultusConfig corresponding net-attach-def firstly
+// Since the MultusConf doesn't have Finalizer, you could delete it as soon as possible and we can't filter it to delete the net-attach-def at first.
+func (pw *podManager) ValidateDelete(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
```

Comment:
this function is empty ? so the comment above does not make sense ? delete them if so

Reply:
yes, deleted